### DISCUSSION TITLE
Welcome to Cronicle Discussions!

### DISCUSSION THREAD

**Discussion by jhuckaby** [jhuckaby/Cronicle#388] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dheise-rheinenergie-trading-com** [comment]
since this was my idea, I can also start to introduce myself. 

My name is Daniel and I work for an energy provider. I have been using Cronicle as a scheduling system for about 1.5 years now. Very successfully, mind you ;-)
Currently we process about 25.000 jobs daily. Our 252 configured events runs smoothly distributed on 3 linux servers (behind a hardware load balancer) and a NAS used for storage.
We use Cronicle to:
- fill up our monitoring system with metrics
- upload and download files to sftp servers
- encode and decode files
- move files around our servers
- run different tasks in databases
- ...
In the last few months, i have greatly expanded our scheduling and implemented a few solutions on and around the cronicle. For example:
- a file event monitor that triggers new events when files are created in defined folders
- start jobs/scripts on windows servers
- start events from our grafana dashboards
- skip special start times for more flexible time management

i hope that cronicle will continue as an active project for a long time and that we will see a v2 really soon. 😄 

so long...

greetings from cologne
Daniel

**Comment by mikeTWC1984** [comment]
wow, 25K events daily. How do you manage logs, are you using default configs? How much space does it take?
Thanks for info btw, I was always wondering what would be practical limits for using  Cronicle. Your post proves  it's a prod-ready tool for large projects

**Reply by dheise-rheinenergie-trading-com** [reply]
On the whole, I use the default configuration. Apart from the settings for the mail server and the certificate files, only the following parameters have been adjusted:
       "debug_level": 2,
       "job_data_expire_days": 30,
       "master_ping_freq": 15,
       "master_ping_timeout": 30,
       "log_event_types": { "get": 0, "put": 0, "head": 0, "delete": 0, "expire_set": 0 },

My data directory currently contains about 5.9 million files and is 45 gb in size. Currently, Cronicle also feels like it can do much more. In the past, we have had up to 30,000-35,000 jobs per day.

Daniel

**Comment by jhuckaby** [comment]
Wow, @dheise-rheinenergie-trading-com, that is really amazing.  I've never pushed Cronicle that hard, not even in testing.  I think you have the largest installation by far 😉

**Reply by dheise-rheinenergie-trading-com** [reply]
And we are certainly not at the end yet. There will most definitely be one or two more jobs to come. ;-)
